src/Plugin.php

```diff
@@ -70,9 +70,11 @@ public static function post_gallery($output = '', $atts) {
     $atts['post_type'] = 'attachment';
     $atts['post__in'] = $atts['include'];
     $layout = !empty($atts['layout']) ? $atts['layout'] : apply_filters('gallerya/default_layout', self::DEFAULT_LAYOUT);
+    $min_image_to_show_nav = apply_filters('min_image_to_show_nav', 5);
```

Comment:
Following the filter only approach is a good idea. This will prevent the user from customizing each gallery 👍. But the filter should be prefixed by the plugin name as in the line above. In addition, the filter name should be plural. 

```diff
- $min_image_to_show_nav = apply_filters('min_image_to_show_nav', 5);
+ $min_images_to_show_nav = apply_filters('gallerya/min_images_to_show_nav', 5);
```